02-Logging/01-Logging-to-a-GSD-file.ipynb

```diff
@@ -230,7 +230,7 @@
     {
```

Comment:
<p>There is no longer any data being printed here. Does that indicate the array is empty, or is there some kind of intentional suppression of the printed output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
The data is there. See it in the source: https://github.com/glotzerlab/hoomd-examples/blob/gsd-flush/02-Logging/01-Logging-to-a-GSD-file.ipynb or scroll down the text area in ReviewNB. The values differ because I reran the notebook and the results are not exactly binary reproducible from run to run.

I include all the rerun results to ensure that the notebooks are self-consistent.